src/otg/assets/schemas/study_locus.json

```diff
@@ -172,43 +172,55 @@
             },
             {
               "metadata": {},
-              "name": "tagVariantId",
+              "name": "variantId",
```

Comment:
I think this is the only field in the locus object that could be nullable: false.